_posts/2020-06-16-hhvm-4.62.markdown

```diff
@@ -0,0 +1,65 @@
+---
+title: "HHVM 4.62"
+layout: post
+author: fred
+category: blog
+---
+
+HHVM 4.62 is released! HHVM 4.57&ndash;4.59 remain supported, as do the 4.32
```

Comment:
The range 4.57-4.59 ought to be 4.57-5.61

Reply:
Fixed + squashed into master push